src/core_plugins/kibana/public/dashboard/dashboard_actions.js

```diff
@@ -0,0 +1,121 @@
+import {
+  EMBEDDABLE_RENDER_FINISHED,
```

Comment:
I would look at redux-actions.  Makes this all more concise: https://www.npmjs.com/package/redux-actions

Reply:
I *really* dislike the action types pattern in the redux world. To expand on Bill's recommendation, using redux-actions allows you to export and use your actions directly in both the reducers and in the containers, so the definition lived directly where it gets used and can be references all over the same way without the need for a separate collection of action types.